static/js/reducers/program_learners.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { fetchJSONWithCSRF } from 'redux-hammock/django_csrf_fetch';
+
+import type { Endpoint } from '../flow/restTypes';
+import { GET } from '../constants';
+import { INITIAL_STATE } from '../lib/redux_rest_constants';
+import { getProgramLearners } from '../lib/api';
+
+export const INITIAL_PROGRAM_LEARNERS_STATE =  {
+  ...INITIAL_STATE,
+  data: [],
+};
+
+export const programLearnersEndpoint: Endpoint = {
+  name: 'programLearners',
+  namespaceOnUsername: true,
+  checkNoSpinner: true,
+  verbs: [GET],
+  getFunc: getProgramLearners,
+  initialState: {},
+  usernameInitialState: INITIAL_PROGRAM_LEARNERS_STATE,
+  fetchFunc: fetchJSONWithCSRF,
```

Comment:
oh, you shouldn't need to specify the `fetchFunc` here since you're specifying the `getFunc`. A different way to do it is to specify `getURL` and let the redux-hammock code figure out how to go about calling `fetch` with that URL, but if you specify the `getFunc` `redux-hammock` will just use that.